subprojects/core/src/integTest/groovy/org/gradle/initialization/CommandLineArgDeprecationIntegrationTest.groovy

```diff
@@ -41,30 +42,33 @@ class CommandLineArgDeprecationIntegrationTest extends AbstractIntegrationSpec {
         'https://github.com/gradle/gradle/issues/1425' | '--recompile-scripts' | RECOMPILE_SCRIPTS_MESSAGE
         'https://github.com/gradle/gradle/issues/3077' | '--no-rebuild'        | NO_REBUILD_MESSAGE
         'https://github.com/gradle/gradle/issues/3077' | '-a'                  | NO_REBUILD_MESSAGE
-        'https://github.com/gradle/gradle/issues/3334' | '--no-search-upward'  | NO_SEARCH_UPWARD_MESSAGE
```

Comment:
Why was this removed?

Reply:
That's removed by you... https://github.com/gradle/gradle/pull/3651

Reply:
Ah, kind of weird that it shows up as change in your PR though.